src/monodroid/jni/external-api.cc

```diff
@@ -1,29 +1,69 @@
-#ifdef WINDOWS
-#include <windef.h>
-#include <winbase.h>
-#include <shlobj.h>
-#include <objbase.h>
-#include <knownfolders.h>
-#include <shlwapi.h>
-#endif
-
-#include "globals.hh"
-#include "timing.hh"
+#include <unistd.h>
+#include <mono/utils/mono-publib.h>
+
+#include "monodroid-glue-external.hh"
+
+constexpr int TRUE = 1;
+constexpr int FALSE = 0;
 
 using namespace xamarin::android;
-using namespace xamarin::android::internal;
+
+static MonodroidExternalAPI *external_api = nullptr;
+
+MONO_API bool
+__init_external_api (MonodroidExternalAPI *api)
```

Comment:
This is a name reserved by the compiler (even if something frequently used by others).  Any particular reason for a double-underscore `__` prefix and not `_monodroid_`?